tmt/steps/report/html.py

```diff
@@ -159,6 +159,10 @@ def go(self):
         environment.filters["basename"] = lambda x: os.path.basename(x)
         template = environment.from_string(HTML_TEMPLATE)
 
+        # Nothing more to do in dry mode
+        if self.opt('dry'):
+            return
+
```

Comment:
This was intentionally after the `self.write()` as this allows to see in the debug log what would happen:

    report
        workdir: /var/tmp/tmt/run-303/plan/report
        how: html
        order: 50
            Write file '/var/tmp/tmt/run-303/plan/report/default/index.html'.
        summary: 2 tests passed
        status: done
        Write file '/var/tmp/tmt/run-303/plan/report/step.yaml'.

The file is not created though, because the `write()` method [takes care](https://github.com/psss/tmt/blob/master/tmt/utils.py#L445) of the `--dry` mode.

Reply:
I see.